homeassistant/components/idasen_desk/__init__.py

```diff
@@ -28,49 +29,92 @@
 _LOGGER = logging.getLogger(__name__)
 
 
+class IdasenDeskCoordinator(DataUpdateCoordinator):
+    """Class to manage updates for the Idasen Desk."""
+
+    def __init__(
+        self,
+        hass: HomeAssistant,
+        logger: logging.Logger,
+        name: str,
+        address: str,
+    ) -> None:
+        """Init IdasenDeskCoordinator."""
+
+        super().__init__(hass, logger, name=name)
+        self._address = address
+        self._expected_connected = False
+
+        self.desk = Desk(self.async_set_updated_data)
+
+    async def connect(self) -> bool:
```

Comment:
Nothing seems to be checking the return value here. Should this return `None` instead?

Reply:
It is checked on line 72